netket/graph/space_group.py

```diff
@@ -426,18 +465,90 @@ def little_group(self, *k: Array) -> PointGroup:
             unit_cell=self.lattice.basis_vectors,
         )
 
-    def _little_group_irreps(self, k: Array, divide: bool = False) -> Array:
+    def little_group_multipliers(self, *k: Array) -> np.ndarray | None:
+        r"""Computes the Schur multiplier associated with the little group
+        given the translations associated with its elements.
+
+        The mutlipliers are given by (Bradney & Cracknell, eqs. 3.7.11-14)
+
+        .. math::
+
+            \mu(S_i, S_j) = exp(-i g_i \cdot w_j)
+
+            g_i  = S_i^{-1} k - k
+
+        and :math:`w_j` is the translation associated with symmetry `S_i`
+
+        Arguments:
+            k: the wave vector in Cartesian axes
+
+        Returns:
+            a square array of the :math:`\mu(S_i, S_j)`
```

Comment:
Then can you add a mention here that you can also return None if all the multipliers are 1 meaning that one needs linear reps, suggesting to look at the function `....`?